testng-core/src/test/kotlin/org/testng/util/StringsTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.testng.util
+
+import org.assertj.core.api.Assertions.assertThat
+import org.testng.annotations.Test
+
+class StringsTest {
+    @Test
+    fun joinEmptyArray() {
+        val emptyArray = arrayOf<String>()
+        assertThat(Strings.join(",", emptyArray)).isEmpty()
```

Comment:
Strings.join should be deprecated and replaced by the jdk api.
Just remove the test.

Reply:
I don't want to mix that change with this PR. I will fix this as a separate PR.